Classes/Neos/Neos/Ui/Domain/Model/Changes/AbstractCreate.php

```diff
@@ -20,6 +22,13 @@
     protected $nodeType;
 
     /**
+     * The referenceData for the rendered node
```

Comment:
Actually, I'd even more prefer if we could not use an array here, but more specifically set the Fusion path as string explicitely